tests/conftest.py

```diff
@@ -11,7 +11,7 @@
 logging.basicConfig(level=logging.DEBUG)
 
 TEST_FOLDER = pathlib.Path(__file__).parent
-FIXTURES_FOLDER = TEST_FOLDER / 'fixtures'
+FIXTURES_FOLDER = TEST_FOLDER / "fixtures"
```

Comment:
@kigawas iiuc we both like `black` but I'd limit this kind of changes on connexion repo :)

Reply:
https://github.com/ioggstream/connexion/pull/3